prinz_h2o/src/main/scala/pl/touk/nussknacker/prinz/h2o/repository/H2OModelRepository.scala

```diff
@@ -1,3 +1,25 @@
 package pl.touk.nussknacker.prinz.h2o.repository
 
-class H2OModelRepository
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.prinz.h2o.H2OConfig
+import pl.touk.nussknacker.prinz.h2o.model.H2OModel
+import pl.touk.nussknacker.prinz.model.{Model, ModelName}
+import pl.touk.nussknacker.prinz.model.repository.ModelRepository
+import pl.touk.nussknacker.prinz.util.repository.client.{RepositoryClient, RepositoryClientFactory}
+import pl.touk.nussknacker.prinz.util.repository.payload.ModelPayload
+
+class H2OModelRepository(implicit val config: H2OConfig)
+  extends ModelRepository with LazyLogging with RepositoryClient {
```

Comment:
please make it more readable when changind code jsut by separatint used traits in this class by 
extends ModelRepository
with LazyLogging
with RepositoryClient